docs/dev/tools/common-errors.md

```diff
@@ -167,6 +167,10 @@ In **JSON** mode, it will resemble the following:
 }
 ```
 
+### Accidental deletion of machines, locations, organizations, or accounts
+
+If you delete your machine, location, organization, or account by mistake, contact [contact@viam.com](mailto:contact@viam.com) immediately - they will try to help but cannot guarantee recovery or restoration.
```

Comment:
```suggestion
If you delete your machine, location, organization, or account by mistake, contact [contact@viam.com](mailto:contact@viam.com) immediately.
They will try to help but cannot guarantee recovery or restoration.
```
[opt] since em dashes are a bit informal, and this sentence can be broken.